src/logic/profiles.ts

```diff
@@ -1,15 +1,40 @@
-import { Entity, Avatar } from '@dcl/schemas'
+import { Profile, ProfileAvatarsItem } from 'dcl-catalyst-client/dist/client/specs/lambdas-client'
+import { normalizeAddress } from '../utils/address'
 
-export function getProfileAvatar(profile: Pick<Entity, 'metadata'>): Avatar {
-  const [avatar] = profile.metadata.avatars
+export function getProfileAvatarItem(profile: Pick<Profile, 'avatars'>): ProfileAvatarsItem {
+  const [avatar] = profile.avatars ?? []
 
   if (!avatar) throw new Error('Missing profile avatar')
 
   return avatar
 }
 
-export function getProfilePictureUrl(baseUrl: string, { id }: Pick<Entity, 'id'>): string {
-  if (!baseUrl) throw new Error('Missing baseUrl for profile picture')
+export function getProfileName(profile: Pick<Profile, 'avatars'>): string {
+  const { name } = getProfileAvatarItem(profile)
 
-  return `${baseUrl}/entities/${id}/face.png`
+  if (!name) throw new Error('Missing profile avatar name')
```

Comment:
So far so good, but we have an open issue regarding profiles being nameless and we still need to define what to do with them (allow them or do not allow them at Catalyst level), just a heads-up to keep the conversation going 👼